pkg/table-rule-selector/selector_test.go

```diff
@@ -52,6 +55,9 @@ var _ = Suite(&testSelectorSuite{
 		{"schema1", "test1", 2, []string{"schema*", "", "schema*", "test*"}},
 		{"t1", "test1", 1, []string{"t*", "test*"}},
 		{"schema1", "abc1", 2, []string{"schema*", "", "schema*", "abc*"}},
+		{"ikj", "ikb", 1, []string{"ik[hjkl]", "ik[!zxc]"}},
+		{"ikh", "iky", 2, []string{"ik[hjkl]", "ik[!zxc]", "ik[f-h]", "ik[!a-ce-g]"}},
```

Comment:
It seems this case still has random failure, maybe caused by the unordered map iteration

Reply:
Yes, I'll change a better way to compare for results in the test.

Reply:
Done.